src/main/java/net/openhft/chronicle/wire/TextWire.java

```diff
@@ -2743,6 +2734,7 @@ public long int64() {
 
             long l = getALong();
             checkRewind();
+            consumePadding(1);
```

Comment:
What is the significance of this call to `consumePadding(1)`?

Reply:
That's the whole point of this change. When a comma supercedes a number, it gets consumed for free, but if there's whitespace before a comma (permitted by all specs) we need to explicitly consume this single comma. It is really "up to one".